src/main/java/mate/academy/exception/AuthenticationException.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.exception;
+
+public class AuthenticationException extends RuntimeException {
+    public AuthenticationException() {

```

Comment:
Please, check in your implementation if you use all these constructors. If not, remove it